.circleci/workflows.yml

```diff
@@ -991,6 +1038,8 @@ commands:
     parameters:
       docker-image-tag:
         type: string
+      run-emulator-docker-image-tag:
```

Comment:
I'd appreciate comments distinguishing this from the above parameter for readers. It's kind of subtle.